test/index.html

```diff
@@ -10,7 +10,7 @@
   <div id="qunit"></div>
   <script src="../node_modules/qunitjs/qunit/qunit.js"></script>
 
-  <script src="../build/temp/ie8/videojs-ie8.min.js"></script>
+  <script src="../build/temp/ie8/videojs-ie8.js"></script>
```

Comment:
Maybe I'll put together a separate PR wrapping all references to the IE8 script with a conditional comment.